runtime/manta-pc/src/lib.rs

```diff
@@ -135,7 +135,7 @@ parameter_types! {
 
 // Configure FRAME pallets to include in runtime.
 impl frame_system::Config for Runtime {
-	type BaseCallFilter = ();
+	type BaseCallFilter = Everything;
```

Comment:
Will this filter out all the extrinsics or none ?

Reply:
`Everything` means it will allow all of extrinsic in our runtime.
Now, we don't filter any extrinsic in `manta-pc`, but `calamari` will.

Reply:
Right, the BaseFilter is for what calls to allow, not disallow, my bad.